src/components/DropdownMenu.tsx

```diff
@@ -70,15 +67,18 @@ export const DropdownMenuItems = forwardRef<
   )
 })
 
-type DropdownMenuItemProps = LinkProps & { className?: string }
+type DropdownMenuItemLinkProps = LinkProps & { className?: string }
 
-export const DropdownMenuItem = forwardRef<
+export const DropdownMenuItemLink = forwardRef<
```

Comment:
I was going to add an additional component in here: `DropdownMenuItem`, which would be a button. I decided to just implement it where its being used for now since it is a bit different style of menu. Leaving this change anyways as we might add a button in the future.